### PR TITLE
COL-700, new SuiteC tool: /lti/dashboard (Impact Studio)

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/COL-700
https://jira.ets.berkeley.edu/jira/browse/COL-702
https://jira.ets.berkeley.edu/jira/browse/COL-701

Includes:
* wire up pieces associated with `LtiConstants.DASHBOARD` (e.g., cartridge.xml)
* database integration for new tool URL
* v0.01 of splash page has basic user info

### CODE REVIEW

File: public/app/dashboard/splash.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="leaderboard-list-container">
```
Comment (paulkerschen) [inline public/app/dashboard/splash.html]: @johncrossman Does this class work for the dashboard and leaderboard both? If so, might be worth renaming it to something more generic and moving into the main app CSS.

File: public/app/dashboard/splash.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  <div class="media">
+    <div class="media-left media-middle">
+      <div class="media-object col-avatar leaderboard-avatar" data-ng-style="{'background-image': 'url(' + me.canvas_image + ')'}"></div>
```
Comment (paulkerschen) [inline public/app/dashboard/splash.html]: @johncrossman Same with `leaderboard-avatar`.

**Comment by paulkerschen** [comment]
Thanks, @johncrossman, looks like an LTI tool to me. One question about CSS class names.

**Comment by johncrossman** [comment]
Thank you, @paulkerschen  I've removed the leaderboard refs, for now. Amendments in.

**Comment by sandeepmjay** [comment]
Looks good. Merging ! @johncrossman

**Pull request merged**